### PR TITLE
 Feat: Add BGP as an option for the Kube_VIP

### PR DESCRIPTION
**Pull request by tvanfleet**
AB#9103388 Add BGP as an option for the kube_vip]

### CODE REVIEW

File: roles/kube_vip/tasks/main.yml [hunk 1]
```diff
@@ -40,13 +40,23 @@
...
     src: kube-vip.yaml.j2
     dest: /etc/kubernetes/manifests/kube-vip.yaml
     owner: root
     group: root
     mode: "0644"
+  when: not kube_vip_bgp_enabled
+
+- name: Upload Kubernetes manifest (if BGP for VIP)
+  ansible.builtin.template:
+    src: kube-vip.yaml.j2
```
Comment (quadewarren) [inline roles/kube_vip/tasks/main.yml]: Would this need to be kube-vip-bgp.yaml.j2?

Comment (tvanfleet) [reply roles/kube_vip/tasks/main.yml]: thank you . fixed.

**Review by PinkMonstera — COMMENTED** [review]
Reviewed

**Review by quadewarren — APPROVED** [review]
LGTM

**Review by c3-geico — APPROVED** [review]
+1

**Pull request merged**